libs/client/src/types.ts

```diff
@@ -13,17 +13,25 @@ export type RequestLog = {
   timestamp: string; // Using string to represent date-time format, but you could also use 'Date' type if you're going to construct Date objects.
 };
 
+export type Metrics = {
+  inference_time: number;
```

Comment:
Based on our convo, it seems that `inference_time` can be `null` in some cases

```suggestion
  inference_time: number | null;
```